Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/WebAppSettingModel.java

```diff
@@ -41,6 +41,7 @@
     private String webAppName = "";
     private String webContainer = "";
 
+    private String slotName = "production";
```

Comment:
add a boolean to determine whether to deploy to a slot.

Reply:
Good, then we will add a checkbox in the UI to indicate deploying to slot or not.